cli/endpoints/online/triton/single-model/mir_gpu_deployment.yml

```diff
@@ -0,0 +1,18 @@
+name: mirsingle4
+type: online
+auth_mode: aml_token
+traffic:
+    etblue: 100
+deployments:
+    -   name: etblue
+        model: azureml:densenet-onnx:1
+        code_configuration:
+            code:
+                local_path: ./code
+            scoring_script: dummy.py
+        environment:
+            name: single-env
+            version: 1
+            docker:
+                image: metricstestacr.azurecr.io/triton-base:3
```

Comment:
FYI. image name string will be updated later once triton image is published to MCR.